client/src/index.js

```diff
@@ -29,12 +30,20 @@ const Routes = () => {
               Status
             </Menu.Item>
           </Link>
+          <Link className="nav-link" to="/become-interpreter">
+            <Menu.Item name="editorials" active={true}>
+              Become Interpreter
+            </Menu.Item>
+          </Link>
         </Menu>
-
         <div>
           <Route path="/" exact component={Home} />
           <Route path="/about/" component={About} />
           <Route path="/status/" component={Status} />
+          <Route
+            path="/become-interpreter/"
```

Comment:
You can name the route "sign-up-interpreter" instead maybe.

Reply:
Changed the name of the Route